easybuild/easyconfigs/c/CGAL/CGAL-4.14.3-iccifort-2020.1.217.eb

```diff
@@ -0,0 +1,26 @@
+name = 'CGAL'
+version = '4.14.3'
+
+homepage = 'https://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient 
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'iccifort', 'version': '2020.1.217'}
```

Comment:
@bartoldeman Boost depends on the impi toolchain, so it's probably better to use impi for CGAL as well.